README.md

```diff
@@ -0,0 +1,62 @@
+# buildfarm perf tests
+
+## Purpose
+
+This packages defined some tests which invokes `perf_test` from Apex's [performance_test](https://gitlab.com/ApexAI/performance_test) package. This test allows you to test performance and latency.
+
+* There is a test for each RMW:
+
+  - CycloneDDS (standalone test)
+  - FastRTPS (standalone test)
+  - rmw_connext_cpp.pn
+  - rmw_cyclonedds_cpp
+  - rmw_fastrtps_cpp
+  - rmw_fastrtps_dynamic_cpp
+  - rmw_opensplice_cpp
+
+##  Build
+
+1.  Install ROS 2 (https://index.ros.org/doc/ros2/Installation/).
+1.  Source the ROS 2 installation (either `/opt/ros/<rosdistro>/setup.bash` if installing from binaries, or `ros2_ws/install/setup.bash` if building from source):
+    1.  `source /opt/ros/eloquent/setup.bash`
+1.  Make a new workspace and clone this repository into it:
+    1.  `mkdir -p performance_ws/src`
+    1.  `cd performance_ws/src`
+    1.  `git clone https://gitlab.com/ApexAI/performance_test`
+    1.  `git clone https://github.com/cottsay/buildfarm_perf_tests/`
+    1.  `cd ..`
+1.  Build the local workspace:
+    1.  `colcon build --cmake-args -DBUILD_TESTING=True`
```

Comment:
I believe that `BUILD_TESTING` is the default, is it not? Please check on your machine to make sure.
```suggestion
    1.  `colcon build`
```